OomDetector.cpp

```diff
@@ -148,21 +148,21 @@ bool OomDetector::isHeuristicOOM(
     int64_t current,
     const MemoryPressure& pressure,
     OomContext& octx) {
-  int threshold = tunables_->get<int>(Tunables::THRESHOLD);
-  int high_threshold = tunables_->get<int>(Tunables::HIGH_THRESHOLD);
-  int high_threshold_duration =
+  const int threshold = tunables_->get<int>(Tunables::THRESHOLD);
+  const int high_threshold = tunables_->get<int>(Tunables::HIGH_THRESHOLD);
+  const int high_threshold_duration =
       tunables_->get<int>(Tunables::HIGH_THRESHOLD_DURATION);
-  float fast_fall_ratio = tunables_->get<float>(Tunables::FAST_FALL_RATIO);
+  const float fast_fall_ratio = tunables_->get<float>(Tunables::FAST_FALL_RATIO);
```

Comment:
Can you wrap the line? I'll put up a clang-format config later.
```
const float fast_fall_ratio = 
    tunables_->get<float>(Tunables::FAST_FALL_RATIO);
```

Reply:
Done